### PR TITLE
add "none" to slot analysis/data type

### PR DESCRIPTION
**Pull request by bmeluch**
From https://github.com/microbiomedata/submission-schema/issues/166, added "none" to AnalysisTypeEnum

### CODE REVIEW

**Comment by pkalita-lbl** [comment]
Sorry for not looking at issue #166 sooner. I just read through it and it's not clear to me if this change needs to be here or in `nmdc-schema`. Hold tight until we get an update in the issue.

**Comment by turbomam** [comment]
What's the case in which the [analysis_type](https://github.com/microbiomedata/nmdc-schema/blob/3e126dbdeb300d601856c19710627a9b9acb7834/src/schema/portal/sample_id.yaml#L33-L46) would be 'none'?

Adding 'none' to a list of categorical values is very controversial. Some terminological resources do it extensively (like NCIT, I think) but I don't think we have done this in the nmdc-schema yet and I would prefer not to start.

The slot is `recommended`, not `required` in nmdc-schema, but it appears to be [required in the submission-schema](https://github.com/microbiomedata/submission-schema/blob/326c606103f7bf0424096d80e3febb29851966cf/src/nmdc_submission_schema/schema/nmdc_submission_schema.yaml#L45870-L45888).

**Comment by pkalita-lbl** [comment]
> What's the case in which the [analysis_type](https://github.com/microbiomedata/nmdc-schema/blob/3e126dbdeb300d601856c19710627a9b9acb7834/src/schema/portal/sample_id.yaml#L33-L46) would be 'none'?

It's outlined in the linked issue #166

File: schemasheets/tsv_in/enums.tsv [hunk 1]
```diff
@@ -6,6 +6,7 @@ AnalysisTypeEnum	metagenomics	placeholder PV descr
 AnalysisTypeEnum	metaproteomics	placeholder PV descr				
 AnalysisTypeEnum	metatranscriptomics	placeholder PV descr				
 AnalysisTypeEnum	natural organic matter	placeholder PV descr				
+AnalysisTypeEnum	none	placeholder PV descr
```
Comment (mslarae13) [inline schemasheets/tsv_in/enums.tsv:9]: @bmeluch 
Change 'none' to "bulk chemistry"

**Comment by mslarae13** [comment]
@pkalita-lbl should this change also be made in NMDC schema?

**Comment by pkalita-lbl** [comment]
> should this change also be made in NMDC schema?

It should **only** be made in `nmdc-schema`. The fact that `submission-schema` was redefining `AnalysisTypeEnum` was a mistake that I fixed recently (which is also why this PR has merge conflicts).

The path forward should be:
* Close this PR
* Make the change in `nmdc-schema`
* When a new version of `nmdc-schema` has been released with that change, update `sheets_and_friends/tsv_in/import_slots_regardless.tsv` in this repo to import the new version

**Pull request closed without merging**